mozregression/cli.py

```diff
@@ -258,7 +259,8 @@ def create_parser(defaults):
     parser.add_argument('--launch',
                         metavar="DATE_OR_REV",
                         help="Launch only one specific build by date (nightly)"
-                             " or changeset (inbound).")
+                             " or changeset (inbound). You can also launch a"
+                             " release by specific release number.")
```

Comment:
Nit: "You can also launch a release by specific release number" => "You can also launch a release using a specific release number"